projects/components/src/filtering/filter/filter-operators.ts

```diff
@@ -78,6 +84,7 @@ export const incompatibleOperators = (operator: FilterOperator): FilterOperator[
   switch (operator) {
     case FilterOperator.In:
     case FilterOperator.Equals:
+    case FilterOperator.NotIn:
       return [
         FilterOperator.In,
```

Comment:
this logic seems off and is not symmetric. `Not in` would return `in` is incompatible, but `in` would not return `not in` as incompatible.

Reply:
Yeah, I am not sure what this is code is doing. Still playing around with it.

Reply:
I'm not 100%, but I believe this is used when adding filters via table controls, so if you do something like add an equals filter it replaces any current one.